client/src/components/buttons/Buttons.stories.tsx

```diff
@@ -19,116 +19,198 @@
+    },
+  },
+};
 
-export const SecondaryPink = (args: LabelsProps) => (
+export const Workflows = (args: LabelsProps) => (
   <>
-    <Button className="btn-outline-rk-pink">{args.text}</Button>
+    <Button className="btn-rk-yellow">{args.text}</Button>
   </>
 );
-SecondaryPink.args = {
+Workflows.args = {
   text: "My Button",
 };
+Workflows.parameters = {
+  docs: {
+    description: {
+      story:
+        'Use the class "btn-rk-yellow" when working on workflow components.',
+    },
+  },
+};
 
-const defaultAction = <Button key="button-main-primary">Main Action</Button>;
-const options = [
-  <Button key="button-a">Option A</Button>,
-  <Button key="button-B">Option B</Button>,
-];
-export const menuWithOptionPrimary = () => (
+export const Informative = (args: LabelsProps) => (
   <>
-    <ButtonWithMenu default={defaultAction} isPrincipal={true} color="rk-green">
-      {options}
-    </ButtonWithMenu>
+    <Button className="btn-info">{args.text}</Button>
   </>
 );
-const defaultActionSecondary = (
-  <Button key="button-a" className="btn-outline-rk-green">
-    Main Action
-  </Button>
-);
-export const menuWithOptionSecondary = () => (
+Informative.args = {
+  text: "Update Version",
+};
+Informative.parameters = {
+  docs: {
+    description: {
+      story:
+        "Use `btn-info` in the context of an Info alert or to enhance user experience by offering context or supplementary actions. It is not associated with critical actions. E.G `Update Version`.",
+    },
+  },
+};
+
+export const Warning = (args: LabelsProps) => (
   <>
-    <ButtonWithMenu
-      default={defaultActionSecondary}
-      isPrincipal={false}
-      color="rk-green"
-    >
-      {options}
-    </ButtonWithMenu>
+    <Button className="btn-warning">{args.text}</Button>
   </>
 );
+Warning.args = {
+  text: "Discard Changes",
+};
+Warning.parameters = {
+  docs: {
+    description: {
+      story:
+        "Use `btn-warning` in the context of a Warning Alert, or to signal caution or potential risks. It is employed for actions that might have consequences but can still be undone or rectified.",
+    },
+  },
+};
 
-const optionsGroupButton = [
-  <Button key="button-x" className="btn-outline-rk-green">
-    Option X
-  </Button>,
-  <Button key="button-y" className="btn-outline-rk-green">
-    Option y
-  </Button>,
-];
-export const roundButtonGroup = () => (
+export const Error = (args: LabelsProps) => (
   <>
-    <RoundButtonGroup>{optionsGroupButton}</RoundButtonGroup>
+    <Button className="btn-danger">{args.text}</Button>
   </>
 );
+Error.args = {
+  text: "Delete Project",
+};
+Error.parameters = {
+  docs: {
+    description: {
+      story:
+        "Use `btn-danger` in the context of a Danger Alert, or when suggesting a high-risk actions action that can lead to critical or irreversible outcomes. (E.G. `Delete Project` or updating a project when it is on a very outdated version not supported anymore by the UI).",
```

Comment:
An additional `actions` slipped in (or `action`, one of the two)
```suggestion
        "Use `btn-danger` in the context of a Danger Alert, or when suggesting a high-risk action that can lead to critical or irreversible outcomes. (E.G. `Delete Project` or updating a project when it is on a very outdated version not supported anymore by the UI).",
```